### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Juanmcuello**
- Fix typos.
- Remove trailing whitespaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -649,7 +649,7 @@ def foo
 end
 ```
 
-On this case, you can see the result of `bar()` everytime when you stops there.
+On this case, you can see the result of `bar()` every time when you stops there.
```
Comment (olleolleolle) [inline README.md:652]:
```suggestion
On this case, you can see the result of `bar()` every time you stop there.
```

How about this?

Comment (Juanmcuello) [reply README.md]: Good catch, I missed that. Fixed.

File: misc/README.md.erb [hunk 2]
```diff
@@ -411,7 +411,7 @@ Emacs support available.
 
 #### Start by method
 
-After loading `debug/session`, you can start debug session with the following methods. They are convinient if you want to specifies debug configrations in your program.
+After loading `debug/session`, you can start debug session with the following methods. They are convenient if you want to specifies debug configurations in your program.
```
Comment (olleolleolle) [inline misc/README.md.erb:414]:
```suggestion
After loading `debug/session`, you can start debug session with the following methods. They are convenient if you want to specify debug configurations in your program.
```

File: misc/README.md.erb [hunk 3]
```diff
@@ -355,7 +355,7 @@ config set no_color true
 
 ### Initial scripts
 
-If there is `~/.rdbgrc`, the file is loaded as an initial scripts which contains debug commands) when the debug session is started. 
+If there is `~/.rdbgrc`, the file is loaded as an initial scripts which contains debug commands) when the debug session is started.
```
Comment (olleolleolle) [inline misc/README.md.erb:358]:
```suggestion
If there is `~/.rdbgrc`, the file is loaded as an initial script (which contains debug commands) when the debug session is started.
```

**Review by olleolleolle — APPROVED** [review]
This is a good upgrade of the README's readability.

Thanks, @Juanmcuello!

**Comment by ko1** [comment]
Thank you!

**Pull request merged**